Ooui/Element.cs

```diff
@@ -238,6 +240,21 @@ public void RemoveAttribute (string attributeName)
             }
         }
 
+        public void SetLocalAttribute (string attributeName, object value)
+        {
+            lock (localAttributes) {
```

Comment:
I am uncertain if `lock` makes sense here. I suspect `lock` is used for attributes to allow for the socket thread to update attributes but local attributes will not be updated by the socket thread.

Reply:
My goal has been for Ooui to be thread safe so I appreciate the lock.